src/corg/developer-environment-setup/getting-the-nuxeo-source-code.md

```diff
@@ -225,75 +225,100 @@ Starting from version 5.9.2, Nuxeo requires Maven <span style="color: rgb(34,34,
 
 ### Getting the Development Branch
 
-{{#> panel type='code' heading='Clone in Read-Only mode'}}
-
-```
-git clone git://github.com/nuxeo/nuxeo.git
-cd nuxeo
-python clone.py master -a
-
-```
-
-{{/panel}}{{#> panel type='code' heading='Clone in Read+Write mode'}}
+{{#> panel type='code' heading='Clone Nuxeo repositories'}}
 
 ```
 git clone git@github.com:nuxeo/nuxeo.git
 cd nuxeo
 python clone.py master -a
-
 ```
 
-{{/panel}} {{#> callout type='note' }}
+{{/panel}}
+
+{{#> callout type='note' }}
 
-*   clone.py can be run with Python 2.7.x but not 3.x versions.
+*   clone.py must be run with Python 2.7 and not Python 3.x versions.
 *   Users of MSysGit should run clone.py with option --no_drive_mapping or -n as MSysGit is not able to access dynamically mounted drives. This option is available as of version 5.7.1.
 
-{{/callout}} {{#> callout type='note' }}
+{{/callout}}
+
+{{#> callout type='note' }}
 
 Contributing to Nuxeo **does not** require cloning Nuxeo source code: see [Contributing to Nuxeo]({{page space='nxdoc' page='contributing-to-nuxeo'}})
 
 {{/callout}}
 
-### Import Nuxeo Source Code in Eclipse
-
-Of course for the dependencies to be resolved in Eclipse they will have to be loaded first in your Maven repository, which you can do by performing at least one build first:
+After having cloned the Nuxeo repository, you need to run a full build in Maven to download all the dependencies.
 
 {{#> panel type='code' heading='Full build Nuxeo without running tests'}}
 
 ```
-export MAVEN_OPTS="-Xmx4096m -Xms1024m -XX:MaxPermSize=512m"
+export MAVEN_OPTS="-Xmx4096m -Xms1024m"
 mvn -DskipTests install -Paddons
-(cd nuxeo-distribution ; mvn -DskipTests install)
+cd nuxeo-distribution
+mvn -DskipTests install
```

Comment:
I'm not familiar with it... Could I modify it directly to `-Paddons,distrib` or we still need somebody to confirm this command?